### DISCUSSION TITLE
How to set application_name on postgresql url

### DISCUSSION THREAD

**Discussion by azcoto** [prisma/prisma#8422] [Q&A] [answered]
I tried to add application_name on url in schema like :

postgresql://postgres:********@localhost:5432/pds?schema=public&**application_name=myappname**

when i query the pg_stat_activity table the application_name in connection url is not added to activity

**Comment by ryands17** [comment, marked as answer]
I'm not sure if this is possible but I will let @pantharshit00 confirm.

**Reply by janpio** [reply]
This is not supported yet, but we are happy to take a feature request about this into account. We recently added something similar for SQL Server I think.

**Reply by azcoto** [reply]
Thank you for the answer. I hope Prisma team can implement this feature it soon.

Because i can append app_name:app_username and use it for populating audit table

**Reply by ryands17** [reply]
@azcoto It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
